docs/release-notes/index.md

```diff
@@ -0,0 +1,267 @@
+---
+navigation_title: "Elastic APM Node.js Agent"
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/nodejs/current/release-notes.html
+  - https://www.elastic.co/guide/en/apm/agent/nodejs/current/release-notes-4.x.html
+---
+
+# Elastic APM Node.js Agent release notes [elastic-apm-nodejs-agent-release-notes]
+
+Review the changes, fixes, and more in each version of Elastic Node.js Java Agent.
+
+To check for security updates, go to [Security announcements for the Elastic stack](https://discuss.elastic.co/c/announcements/security-announcements/31).
+
+% Release notes includes only features, enhancements, and fixes. Add breaking changes, deprecations, and known issues to the applicable release notes sections.
+
+% ## version.next [elastic-apm-nodejs-agent-versionext-release-notes]
+% **Release date:** Month day, year
+
+% ### Features and enhancements [elastic-apm-nodejs-agent-versionext-features-enhancements]
+
+% ### Fixes [elastic-apm-nodejs-agent-versionext-fixes]
+
+## 4.11.1 [elastic-apm-nodejs-agent-4-11-1-release-notes]
+**Release date:** March 3, 2025
+
+### Features and enhancements [elastic-apm-nodejs-agent-4-11-1-features-enhancements]
+
+* Update base image of alpine in `Dockerfile` to version `3.21.3`. ([#4465](https://github.com/elastic/apm-agent-nodejs/pulls/4465))
+* Test FIPS 140 compliance. ([#4441](https://github.com/elastic/apm-agent-nodejs/pulls/4441))
```

Comment:
These were marked as `Chore` in the AsciiDoc changelog. I'm not sure if they belong in `Features and enhancements` or `Fixes`.  cc @KOTungseth for guidance on how to handle chore-like updates in release notes across Elastic.

Reply:
Chores typically include code quality improvements, stability enhancements, developer experience improvements, and consistency improvements, which is why I bucketed them under `Features and enhancements`.

If there are specific actions that users need to take to update the base image of alpine in `Dockerfile` or considerations for FIPS testing, we should be including that content in the narrative docs, not the release notes.